libs/surfaces/console1/console1.cc

```diff
@@ -697,7 +725,19 @@ Console1::set_current_stripable (std::shared_ptr<Stripable> r)
 			_current_stripable->mapped_control (EQ_Enable)->Changed.connect (
 			  stripable_connections, MISSING_INVALIDATOR, std::bind (&Console1::map_eq, this), this);
 		}
-
+#ifdef MUXBUS
```

Comment:
MUXBUS ? :)

Reply:
Wonder why my last test in MIXBUS worked....
Obvoiusly I did not test enough.